scout/server/blueprints/institutes/controllers.py

```diff
@@ -638,28 +648,49 @@ def get_genome_build(variant_case_obj):
     return "37"
 
 
-def get_hgvs(gene_obj):
-    """Analyse gene object
+def get_hgvs(gene_obj: Dict) -> Tuple[str, str, str]:
+    """Analyse gene object for hgvs info
     Return:
-       (hgvs_nucleotide, hgvs_protein)"""
+       (canonical_transcript, hgvs_nucleotide, hgvs_protein)"""
     hgvs_nucleotide = "-"
     hgvs_protein = ""
 
     transcripts_list = gene_obj.get("transcripts")
     for transcript_obj in transcripts_list:
         if transcript_obj.get("is_canonical") is True:
+            canonical_transcript = transcript_obj.get("transcript_id")
             hgvs_nucleotide = str(transcript_obj.get("coding_sequence_name"))
             hgvs_protein = str(transcript_obj.get("protein_sequence_name"))
-    return (hgvs_nucleotide, hgvs_protein)
+    return (canonical_transcript, hgvs_nucleotide, hgvs_protein)
+
+
+def get_annotations(gene_symbols: List, gene_annotations: List) -> List:
```

Comment:
The thing with the tilde (place holding for gnomAD frequency) is that each element of the frequency cell has its own frequency tooltip, the Local badge the local freq, the clinvar one the clinvar things. We could join them all into one instead, perhaps with shortened badges like dunno `L` and `C` to get all into a safely one-liner. For now, I've updated the justification. Looks neat on the gene-variants page, and at the very least not worse on the variantS. 

![Screenshot 2023-06-12 at 12 18 29](https://github.com/Clinical-Genomics/scout/assets/758570/d9c6b97a-ba49-4eaa-b61f-2a1c1d8e2cd3)

![Screenshot 2023-06-12 at 12 21 30](https://github.com/Clinical-Genomics/scout/assets/758570/336ada74-3e23-47a6-abe5-c05e62e547c2)

I'll test it on stage as well to see there are is nothing weird going down when more stuff is added.

Reply:
Hm, I had to fiddle a little with spacing and cell widths, and think it ended up pretty ok:
![Screenshot 2023-06-12 at 13 44 47](https://github.com/Clinical-Genomics/scout/assets/758570/652ec534-b590-4390-a248-cae4654a7ff4)
It's better for large screens, not worse down to 1400 width, and marginally worse down to 1200. After that, I think the old was better, but so many other things look lousy anyway.

Reply:
Ah, but I managed to find an issue with missing data when running on stage. Nice! 🙄

Reply:
Heh, it was actually more data rather than missing. :blush:
Fixed now. I think its good to go:

![Screenshot 2023-06-12 at 14 13 14](https://github.com/Clinical-Genomics/scout/assets/758570/a27430f7-b58f-4ba9-8a64-1c5e4d6ebfac)

Reply:
> Ah, but I managed to find an issue with missing data when running on stage. Nice! 🙄

Let's merge. We can test again the release branch!